karma.conf.js

```diff
@@ -11,6 +11,8 @@ module.exports = config =>
     browsers: ['Chrome'],
 
     karmaTypescriptConfig: {
-      compilerOptions: tsconfig.compilerOptions
+      compilerOptions: {
+        ...tsconfig.compilerOptions
```

Comment:
I don't understand this change. Why create a new object here?

Reply:
Ah, sorry. It was an artifact from an earlier change. I'll remove.